test/com/ableton/VirtualEnvTest.groovy

```diff
@@ -25,8 +25,6 @@ class VirtualEnvTest extends BasePipelineTest {
     assertNotNull(script)
     script.env = ['BUILD_NUMBER': 1, 'JOB_BASE_NAME': 'mock']
 
-    helper.registerAllowedMethod('deleteDir', [], JenkinsMocks.deleteDir)
```

Comment:
Are these lines no longer required in v0.8?

Reply:
Correct, they were removed in https://github.com/AbletonDevTools/jenkins-pipeline-mocks/pull/38 (which was actually reviewed by you 😉).